.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+## Deprecation warning ⚠️
+
+The Timber theme is no longer being maintained by Shopify.  Issues will not be addressed by this
```

Comment:
You working on a typewriter? Double space after periods in a few of these .md files. :trollface: